cmd/bucket-metadata-sys.go

```diff
@@ -600,8 +596,8 @@ func (sys *BucketMetadataSys) Initialized() bool {
 }
 
 // Loads bucket metadata for all buckets into BucketMetadataSys.
-func (sys *BucketMetadataSys) init(ctx context.Context, buckets []BucketInfo) {
-	count := 100 // load 100 bucket metadata at a time.
+func (sys *BucketMetadataSys) init(ctx context.Context, buckets []string) {
+	count := globalEndpoints.ESCount()
```

Comment:
```suggestion
	count := globalEndpoints.ESCount() * 10
```

? would this work? - we can add staggered latencies of 1ms to 10ms between each loaders() to avoid stamping herd?

Reply:
otherwise this will slow down the startup of smaller setups significantly.